src/media/icon.story.jsx

```diff
@@ -25,13 +25,6 @@ storiesOf('Icon', module)
 			</WithNotes>
 		</div>
 	))
-	.add('Inverted', () => (
```

Comment:
I saw that the signature of this component changed to no longer support inverted, We should make sure a) we update versions for breaking changes when we do this and remove depricated stories